libs/contact/driver/zendesk/src/zendesk.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DaffContactDriver } from '@daffodil/contact';
+import { DaffContactZendeskService } from './zendesk.service';
+import { DaffContactZendeskConfig, DaffContactZendeskConfigToken } from './config';
+import { DaffContactZendeskConfigService } from './config.service';
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class DaffContactZendeskDriverModule {
+  static forRoot(
+    config: DaffContactZendeskConfig
+  ): ModuleWithProviders<DaffContactZendeskDriverModule> {
+    return {
+      ngModule: DaffContactZendeskDriverModule,
+      providers: [
+        { provide: DaffContactZendeskConfigToken, useValue: config },
```

Comment:
No, because each module's driver configuration will be unique, and I don't want to risk cross-pollination where someone includes two drivers and accidentally mixes the tokens.